lamindb/core/_sync_git.py

```diff
@@ -24,8 +24,7 @@ def get_git_repo_from_remote() -> Path:
         f"running outside of synched git repo, cloning {repo_url} into {repo_dir}"
     )
     result = subprocess.run(
-        f"git clone --depth 10 {repo_url}.git",
-        shell=True,
```

Comment:
I'm pretty sure the shell=True is there for a purpose. 🤔

I have to admit that I never attempted to understand when it's necessary and when not; but I remember that some things didn't work without it. I'd be astonished if this works without it (meaning I added it "just for fun").

Is it possible that the Mac environment is at fault and that's why this might not be caught by CI? I will see if I can still locally run the tests for this.

Reply:
I don't really see why `shell=True` would be necessary here although I also had weird quirks with it in the past. Generally, `shell=True` is necessary when you're using shell features such as wild cards, piping, and other things.

Reply:
Test works! No worries about the arg list. Can stay as is.